tests/integration-tests/configs/common/common.yaml

```diff
@@ -649,19 +579,19 @@ storage:
     dimensions:
       - regions: ["eu-south-1"]
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
-        oss: {{ common.OSS_ONE_PER_DISTRO }}
+        oss: ["rhel8"]
         schedulers: ["slurm"]
       - regions: ["ap-southeast-1"]
         instances: {{ common.INSTANCES_DEFAULT_ARM }}
```

Comment:
Why do we need ARM and X86 to test a feature regarding AWS?

Reply:
The idea is to have at least one FSx test per OS, to verify Lustre client behaviour. Anyway I agree we can consolidate more.

Reply:
Done in https://github.com/aws/aws-parallelcluster/pull/5201